tests/unit/initializers/setup-session-test.js

```diff
@@ -19,6 +19,7 @@ describe('setupSession', () => {
       register() {},
       injection() {}
     };
+    Ember.testing = true;
```

Comment:
Not really feelin' this, but `tearDownContext()` in the `new-style-helpers-test` blows this test up.

Reply:
Seems fine to me. I do dislike that Ember-test-helpers has to do that...